index.bs

```diff
@@ -379,20 +379,66 @@ If a document's [=document/embedder policy=] is "`require-corp`", then any docum
 [[#cascade-vs-require]]).
 
 To <dfn abstract-op lt="process navigation response">check a navigation response's adherence to its
-embedder's policy</dfn> given a [=response=] (|response|), and a target [=browsing context=]
-(|target|), execute the following steps, which will return "`Allowed`" or "`Blocked`" as
-appropriate:
+embedder's policy</dfn> given a [=request=] (|request|), a [=response=] (|response|), and a target
+[=browsing context=] (|target|), execute the following steps, which will return "`Allowed`" or
+"`Blocked`" as appropriate:
 
-1.  Let |response policy| be the result of [$parse header|obtaining an embedder policy$] from
+1.  Return "`Allowed`" if |target| is not a [=child browsing context=].
+
+2.  Let |response policy| be the result of [$parse header|obtaining an embedder policy$] from
     |response|.
 
-2.  Return "`Allowed`" if any of the following statements are true:
+3.  Let |parent policy| be |target|'s [=container document=]'s [=document/embedder policy=].
+
+4.  If |parent policy|'s [=embedder policy/report only value=] is "`require-corp`" and
+       |parent policy|'s [=embedder policy/report only reporting endpoint=] is not null and
+       |response policy|'s [=embedder policy/value=] is "`unsafe-none`":
+
+    1. Let |blocked url| be the first URL of |request|'s [=request/URL list=].
+
+    2. Set |blocked url|'s [=url/username=] to the empty string, and its [=url/password=] to `null`.
+
+    3. Set |serialized blocked url| be the result of executing the
+       [URL serializer](https://url.spec.whatwg.org/#concept-url-serializer) on |blocked url|
+       with the |exclude fragment flag| set.
+
+    4. Let |body| be a new object containing the following properties with keys:
+
+        * key: "`blocked`", value: |serialized blocked url|.
+        * key: "`child cross-origin-embedder-policy`", value: |response policy|'s
+               [=embedder policy/value=].
+        * key: "`child cross-origin-embedder-policy (report only)`", value: |response policy|'s
+               [=embedder policy/report only value=].
+
+    5. [Queue](https://w3c.github.io/reporting/#queue-report) |body| as "`coep-navigation`" for
+       |parent policy|'s [=embedder policy/report only reporting endpoint=] on |target|'s
+       [=container document=].
+
+5.  If |parent policy|'s [=embedder policy/value=] is "`require-corp`" and |child policy|'s
+       [=embedder policy/value=] is "`unsafe-none`":
+
+    1.  If |parent policy|'s [=embedder policy/reporting endpoint=] is not null:
+
+        1. Let |blocked url| be the first URL of |request|'s [=request/URL list=].
```

Comment:
"first URL of" => request's URL, just like the last patch.

Reply:
Done.